src/domain/explorer.ts

```diff
@@ -0,0 +1,52 @@
+export type ExplorerType = 'Blockstream' | 'Testnet' | 'Mempool' | 'Nigiri' | 'Custom';
+
+export interface ExplorerURLs {
+  type: ExplorerType;
+  webExplorerURL: string;
+  explorerURL: string; // most of the time webExplorerURL + '/api'
+  websocketExplorerURL: string; // ws:// or wss:// endpoint
+  batchServerURL?: string;
+  wsURL?: string;
+}
+
+const VULPEM_ELECTRS_BATCH_TESTNET = 'https://electrs-batch-testnet.vulpem.com';
+
+export const BlockstreamExplorerURLs: ExplorerURLs = {
+  type: 'Blockstream',
+  webExplorerURL: 'https://blockstream.info/liquid',
+  explorerURL: 'https://blockstream.info/liquid/api',
+  batchServerURL: 'https://electrs-batch-blockstream.vulpem.com',
+  websocketExplorerURL: 'wss://blockstream.info/liquid/electrum-websocket/api',
```

Comment:
This comment says the url shouldn't have `/api` https://github.com/Blockstream/esplora/issues/428#issuecomment-1357090919